src/test/kotlin/com/ampnet/identityservice/controller/AuthorizationControllerTest.kt

```diff
@@ -73,6 +74,10 @@ class AuthorizationControllerTest : ControllerTestBase() {
             val response: AccessRefreshTokenResponse = objectMapper.readValue(result.response.contentAsString)
             verifyAccessRefreshTokenResponse(response)
         }
+        verify("User is created") {
+            val user = userRepository.findByAddress(ADDRESS.toString())
+            assertThat(user).isNotNull
+        }
     }
```

Comment:
Add a test case for the existing user and verify that the new user is not created.